### DISCUSSION TITLE
[GitHub Mobile][Android][Language] - Some suggestions on Chinese translation

### DISCUSSION THREAD

**Discussion by francis-du** [community/community#74] [Mobile]
---

Platform: Android 9
Version: v1.3.10
Feature: Language Switch
Feedback: Francis Toh

--- 

When I switched to the Simplified Chinese interface, I found that several translations were not accurate enough or not localized enough.So the feedback is as follows:

- `Direct Mentions` should be translated into  `与我相关`.

- `Share Feedback` should be translated into  `发送反馈` or `反馈`.

- `Sign Out` should be translated into  `退出登录` or `退出`.

![github](https://user-images.githubusercontent.com/25944814/99692791-a465f000-2ac5-11eb-8525-08c3df043e04.jpg)

- `Follow system` should be translated into  `跟随系统`.

![githb](https://user-images.githubusercontent.com/25944814/99693654-95cc0880-2ac6-11eb-8874-51dc0dca43eb.jpg)

- I think the `Issues` can be translated into `问题` or without translation.

![issue](https://user-images.githubusercontent.com/25944814/99694365-4f2ade00-2ac7-11eb-961e-89ab57850f43.jpg)

- `Starred` should be translated into  `星标`. It just delete `已`,

![star](https://user-images.githubusercontent.com/25944814/99695011-01fb3c00-2ac8-11eb-8b94-a91a8eab0356.jpg)

Here is all my  feedback.

Regrads,
Francis.

**Comment by candyho** [comment]
Hey thanks for your feedback on Simplified Chinese translation. Re the proposed translations for `Issues`, `问题`sounds more like "questions" to me rather than "Issues". Not all the "Issues" are necessarily "questions", the current translation sounds closer to "topic", which I think aligns with the term "Issues" better. What do you think?

I will reflect the rest of your suggestions to our localisation team to further improve the translation quality. Thanks again!

**Reply by xiaopangju** [reply]
However, the word "issue" is a proper noun for Chinese users, so there is no need to translate it specially. There is no obstacle to understanding in retaining the original text.

**Comment by francis-du** [comment]
I also think that it is not appropriate to translate to `question`, I think it is not necessary to translate.

**Comment by xiaopangju** [comment]
I think Follow system should translate to 跟随系统 too.
